guides/source/active_job_basics.md

```diff
@@ -388,6 +389,24 @@ class GuestsCleanupJob < ApplicationJob
 end
 ```
 
+As described in https://guides.rubyonrails.org/active_record_callbacks.html#halting-execution,
+A callback that `throws :abort` will not execute any following callbacks. You can use this
```

Comment:
```suggestion
A callback that `throws :abort` will halt the callback chain, preventing following before and around callbacks from being called and the event from being triggered. You can use this
```

Reply:
I went with a less verbose version in f0dca63915a9ce1ecdf6f9207edbb63ee93a26b1

Appreciate the feedback @bensheldon 🙏🏼, super happy to iterate if you have more suggestions.